auslib/db.py

```diff
@@ -975,13 +976,15 @@ def select(self, where=None, transaction=None, **kwargs):
             ret.append(row)
         return ret
 
-    def insert(self, changed_by, transaction=None, dryrun=False, **columns):
+    def insert(self, changed_by, transaction=None, dryrun=False, change_type=None, **columns):
```

Comment:
Unfortunately, you'll have to look this up in "columns" instead of adding it its own argument. This method overrides a base class one, so the interface needs to be kept the same. You can see a similar example at https://github.com/mozilla/balrog/blob/master/auslib/db.py#L1501. This probably means you don't need to add it to base_columns yourself either.